enterprise/views.py

```diff
@@ -270,12 +273,30 @@ def get(self, request):
             raise Http404
 
         item = 'course' if course_id else 'program'
+
+        if course_id:
+            try:
+                catalog_api_client = CourseCatalogApiServiceClient()
```

Comment:
Should pass in the enterprise_customer.site to CourseCatalogApiServiceClient(). If you move line 286 above this block it would like like this:
```
        enterprise_customer = consent_record.enterprise_customer
        item = 'course' if course_id else 'program'
        if course_id:
            try:
                catalog_api_client = CourseCatalogApiServiceClient(enterprise_customer.site)
            except ImproperlyConfigured:
                raise Http404
```

Reply:
added enterprise_customer.site to courseCatalogApiServiceClient call